### PR TITLE
Source btn

### PR DESCRIPTION
**Pull request by eugene0928**

### CODE REVIEW

File: scenes/source.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import {Scene} from "grammy-scenes";
+import {BotContext} from "#types/context";
+import HLanguage from "#helper/language";
+import {HReplace} from "#helper/replacer";
+
+const scene = new Scene<BotContext>('Source');
```
Comment (toshmukhamedov) [inline scenes/source.ts:6]: I don’t think we need a scene for just sending a message

File: translate/uz.json [hunk 2]
```diff
@@ -66,5 +67,7 @@
     "isha": "🌃 Xufton $state",
     "save": "✅ Saqlash"
   },
-  "enter": "Kirish"
+  "enter": "Kirish",
+  "source": "Manba",
+  "sourceMessage": "_Namoz vaqtlari:_ [islom.uz]($siteLink)\n_Hadislar to'plami:_ [hadislar]($sourceLink)"
```
Comment (toshmukhamedov) [inline translate/uz.json:72]:
```suggestion
  "sourceMessage": "_Namoz vaqtlari:_ [islom.uz]($siteLink)\n_Hadislar to'plami:_ [Riyozus solihiyn]($sourceLink)"
```

File: translate/uz.json [hunk 3]
```diff
@@ -29,7 +29,8 @@
     "🌍 Joylashuvni o'zgartirish",
     "🍽 Ro'za",
     "🔔 Xabarnomani o'zgartirish",
-    "📊 Statistika"
+    "📊 Statistika",
+    "\uD83D\uDCDA Manba"
```
Comment (Xayrulloh) [inline translate/uz.json:33]: what is this

File: scenes/source.ts [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+scene.step(async (ctx) => {
+    let sourceReplyMessage = HLanguage('sourceMessage')
+    const regionTimeLink = 'https://islom.uz/vaqtlar/' + ctx.user.regionId + '/' + (new Date().getMonth() + 1);
```
Comment (Xayrulloh) [inline scenes/source.ts:10]: 12 oy keb qosa nma bo'ladi 🤔

File: scenes/source.ts [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+    sourceReplyMessage = HReplace(
```
Comment (Xayrulloh) [inline scenes/source.ts:12]: wuni optimizatsiya qiliw iloji bormikn

**Review by Xayrulloh — CHANGES_REQUESTED** [review]

File: scenes/source.ts (hunk 1 above)
Comment (eugene0928) [reply scenes/source.ts:6]: it is just for future features, if @Xayrulloh adds smth

File: translate/uz.json (hunk 2 above)
Comment (eugene0928) [reply translate/uz.json:72]: ok

File: translate/uz.json (hunk 3 above)
Comment (eugene0928) [reply translate/uz.json:33]: for icon

File: scenes/source.ts (hunk 4 above)
Comment (eugene0928) [reply scenes/source.ts:10]: sanoq 0dan shu uchun max 11 keladi

File: scenes/source.ts (hunk 5 above)
Comment (eugene0928) [reply scenes/source.ts:12]: bu prosta str ku, nmasini optimize qimoqchimiz

**Review by toshmukhamedov — APPROVED** [review]

**Review by Xayrulloh — APPROVED** [review]

**Pull request merged**